helm/values.yaml

```diff
@@ -147,13 +243,31 @@ daskhub:
 
           c.KubeSpawner.pre_spawn_hook = pre_spawn_hook
 
+        # it is the spawner post stop hook, not related to the notebook lifecycle
+        # we don't need it
+        post_stop_hook: |
```

Comment:
We don't need it here, I will remove it unless we want to keep it here for fun cc @TomAugspurger